articles/creating-your-first-google-web-story/index.md

```diff
@@ -0,0 +1,276 @@
+
+**Control**
+Contrary to social app stories with Web stories, you're in control of how long the story should be on the Web and not having to worry that your beautifully crafted story would expire after 24 hours. The publisher host Web stories, which means the content, the copyright, and rules depend on the publisher's terms of choice.
+
+**Earn money**
+In almost all social app stories, creators don't earn when a story has an ad, but this is different for Web stories. Creators can earn money for ads on the story pages, as well as brand deals, native advertising, and so on.
+
+**Accessibility**
+Since Web stories live on the web, they don't just live in a single app but can be accessed anywhere. Technically Web stories are Web pages under the hood, which makes them get indexed by search engines.
+
+### Creating a Web Story
+
+Creating a Web Story is flexible depending on your needs and expertise. We can create Web Stories using Code to create a story with custom functionalities and NoCode using drag-and-drop tools.
+
+#### Using NoCode
+
+There are drag-and-drop tools like [MakeStories](https://makestories.io/), [Web Stories for WordPress](https://wordpress.org/plugins/web-stories/), [Newsroom AI](https://www.nws.ai/), and [more](https://amp.dev/documentation/tools/?format=stories) that are used to create a story and then embed it into a website. For the sake of this article, MakeStories editor is going to be used.
+
+![MakeStories website](engineering-education/creating-your-first-google-web-story/makestories.png)
+
+Login or Sign-up on MakeStories [here](https://makestories.io/)
+
+After logging or signing up, create a story by clicking on “CREATE NEW STORY”
+
+![MakeStories dashboard](engineering-education/creating-your-first-google-web-story/makestories2.png)
+
+Fill in the details for the story that would be created.
+
+![MakeStories dashboard](engineering-education/creating-your-first-google-web-story/makestories1.png)
+
+The studio should look like this:
+
+![MakeStories dashboard](engineering-education/creating-your-first-google-web-story/makestories3.png)
+
+Pick any of these: Color, Gradient, Texture, Image, Video, GIFs, Overlay, Animation, and add it to the slide
+
+![MakeStories dashboard](engineering-education/creating-your-first-google-web-story/makestories5.png)
+
+Add text to the slide by clicking on the “T” icon and dragging any typography or font combination.
+
+![MakeStories dashboard](engineering-education/creating-your-first-google-web-story/makestories8.png)
+
+Next, add a new slide by clicking on the plus “+” icon below the slide.
+Then add a background image to fill the slide.
+
+![MakeStories dashboard](engineering-education/creating-your-first-google-web-story/makestories6.png)
+
+After editing the story, then publish it by clicking the “PUBLISH” button and “publish it as a Web Story”.
```

Comment:
“PUBLISH” use ``` ` ``` to highlight content rather than quotations.

Reply:
Done